packages/gatsby-source-shopify/src/nodes.js

```diff
@@ -16,6 +16,7 @@ const PRODUCT_OPTION = `ProductOption`
 const PRODUCT_VARIANT = `ProductVariant`
 const SHOP_POLICY = `ShopPolicy`
 const PRODUCT_TYPE = `ProductType`
+const PAGE = `PAGE`
```

Comment:
No need for all caps

```suggestion
const PAGE = `Page`
```